### PR TITLE
#162234039 Display featured author on the home page

### PR DESCRIPTION
**Pull request by PriscillaSam**
#### What does this PR do?
Create a section on the homepage to display the author of the week.

#### Description of Tasks to be completed
- write actions, reducers and sagas to handle API requests and update the store
- write featured author component

#### How can this be manually tested?
- on the terminal run `git clone https://github.com/andela/haven-ah-frontend.git`.
- run `git checkout ft-author_of_the_week-162234039`.
- run `npm install` to install the npm packages.
- run `npm run start:dev` to start the server. The homepage should open up on `localhost:8080`.
#### Screenshots if applicable
screen shot 2018-12-04 at 2 02 31 pm

#### Relevant Pivotal Tracker stories
#162234039

### CODE REVIEW

**Comment by wiztemple** [comment]
Nice work @PriscillaSam

**Comment by ucheg6** [comment]
Good job @PriscillaSam 👍

File: .eslintrc [hunk 1]
```diff
@@ -78,5 +78,17 @@
             "ClassDeclaration": false
           }
         }]
+      },
+      "settings": {
+        "import/resolver": {
+          "node": {
+            "extensions": [
+              ".js",
+              ".jsx",
```
Comment (TheoOkafor) [inline .eslintrc:87]: Awesome

File: public/styles/responsive.scss [hunk 2]
```diff
@@ -18,79 +17,139 @@ $hover-black: rgba(0,0,0,0.2);
...
+  }
+
+  .heading-primary-main {
+     font-size: 20px !important;
+  }
+
+  .heading-primary-sub {
+    font-size: 15px !important;
+    }
+
```
Comment (TheoOkafor) [inline public/styles/responsive.scss]: These can be removed; `.header `has been changed to `.banner`

File: public/styles/App.scss [hunk 3]
```diff
@@ -3,24 +3,27 @@
...
-$facebook-blue: #4267b2;
+$facebook-blue: rgb(42, 84, 168);
 $hover-black: rgba(0,0,0,0.2);
 
 body {
     padding: 0;
     margin: 0;
-    font-family: $heading-font-family;
-    font-family: 'Noto Serif', sans-serif;
+    font-family: $body-content-font;
```
Comment (seyi-adeleke) [inline public/styles/App.scss:23]: Does this take the P/O's feedback about fonts into consideration?

File: src/App.jsx [hunk 4]
```diff
@@ -1,8 +1,10 @@
 import React, { Component } from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
 import { Provider } from 'react-redux';
-import Routes from './routes';
 import 'bulma/css/bulma.min.css';
+
+import Routes from './routes';
```
Comment (seyi-adeleke) [inline src/App.jsx:5]: absolute imports should come before relative imports. The linter didn't flag this?

File: webpack.config.common.js [hunk 5]
```diff
@@ -36,10 +35,12 @@ module.exports = {
   devServer: {
     historyApiFallback: true,
   },
+  resolve: {
+    extensions: ['.js', '.jsx', '.css']
```
Comment (seyi-adeleke) [inline webpack.config.common.js:39]: why?

File: webpack.config.dev.js [hunk 6]
```diff
@@ -7,4 +8,11 @@ module.exports = merge(common, {
   devServer: {
     contentBase: './dist',
   },
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env': {
+        API_URL: JSON.stringify('http://localhost:5000/api/v1')
```
Comment (seyi-adeleke) [inline webpack.config.dev.js:14]: Put these in a .env and see if it works. Not an issue if it doesn't

File: webpack.config.common.js (hunk 5 above)
Comment (PriscillaSam) [reply webpack.config.common.js:39]: This is so we don't have to add extensions when we import `.js` and `.jsx` files

File: src/App.jsx (hunk 4 above)
Comment (PriscillaSam) [reply src/App.jsx:5]: No. The linter didn't flag this. I'd fix it.

File: public/styles/responsive.scss (hunk 2 above)
Comment (PriscillaSam) [reply public/styles/responsive.scss]: Done @TheoOkafor .

File: webpack.config.dev.js (hunk 6 above)
Comment (PriscillaSam) [reply webpack.config.dev.js:14]: I don't understand what you mean here.

File: public/styles/App.scss (hunk 3 above)
Comment (PriscillaSam) [reply public/styles/App.scss:23]: @seyi-adeleke, yes it does.

**Review by seyi-adeleke — APPROVED** [review]

**Pull request merged**